subprojects/build-events/src/integTest/groovy/org/gradle/build/event/BuildEventsParallelIntegrationTest.groovy

```diff
@@ -33,7 +36,7 @@ class BuildEventsParallelIntegrationTest extends AbstractIntegrationSpec {
     }
 
     def "event handling does not block other work"() {
-        buildFile << """
+        buildFile("""
```

Comment:
💅 Parentheses are unnecessary here

Reply:
> 💅 Parentheses are unnecessary here

Ok, this seems to be a prevalent style in our tests.